.github/workflows/e2e.yml

```diff
@@ -3,10 +3,14 @@ on:
   push:
     branches: [main]
   pull_request:
-    branches: [main]
+    types: [labeled]
+
+env:
+  NODE_ENV: testing
 
 jobs:
   test:
+    if: ${{ (github.event.label.name == 'e2e') || (github.ref == 'refs/heads/main') }}
```

Comment:
@adamkudrna @pavelklibani @crishpeen There is also a question about how we should name the label which will allow running the visual tests. -> `e2e`, `visual tests`, `visual testing`, `visual regression testing`, etc...

Reply:
`visual tests` sounds the most intuitive to me.

Reply:
But — you have `e2e` everywhere in the code. Therefore I'd either use `e2e` as the primary label and `visual tests` as its more intuitive alias, or change `e2e` to `visual tests` in the code.

(Is it really end-to-end after all…? I imagine something even more sophisticated than "just" checking visual snapshots…)

Reply:
The problem is that the "visual regression testing" is part of the end-to-end testing. And using Playwright we can achieve both. We can test our demo pages against snapshots and also we can test our demo pages against predefined use cases using clicking and testing visibility or existence of some elements.

So for me is the general name the "end-to-end testing" so I would rather use this naming. But in this time being we are only running visual regression testing so I acknowledge that the "visual testing" is more suitable right now.

In the future, the both will be runned inside the same docker and by the same pipeline. So I can rename it to "visual tests" right now everywhere and when it comes to use some click-to-click and step-by-step approach I will rename it to "e2e". Agreed?

Reply:
We will discuss this at our technical meeting and change this later.